.github/workflows/build.yml

```diff
@@ -0,0 +1,87 @@
+name: Build, Test and Deploy
+
+on:
+  push:
+    branches:
+      - master
+      - beta
+  pull_request:
+
+env:
+  TZ: 'Europe/Paris'
+
+jobs:
+  lint:
+    name: Linting
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - name: Cancel previous running workflows
+        uses: fkirc/skip-duplicate-actions@master
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v1
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: install dependencies
+        run: yarn install --frozen-lockfile --non-interactive --production=false
+      - name: Lint commit message
+        uses: wagoid/commitlint-github-action@v2
+      - name: lint Javascript
+        run: yarn lint
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    needs: [lint]
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v1
+      - name: Cache node_modules
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Start docker container
+        run: docker-compose up -d
+      - name: Send coverage
+        uses: paambaati/codeclimate-action@v2.7.4
+        env:
+          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
```

Comment:
Where is used the `CC_TEST_REPORTER_URL` to download the reporter we want (like it was done with TravisCI)?

Reply:
I don't understand what you mean. I looked into CC doc there is no `CC_TEST_REPORTER_URL`. I don't know for what it was used for 🤷  ?
Can you tell me a little bit more 🤔  ?
What do you mean by downloading the reporter we want ?